app/models/recipe.py

```diff
@@ -31,12 +33,15 @@ class Recipe(db.Model, ModelMixin):
     updated_at: orm.Mapped[datetime] = orm.mapped_column(default=datetime.utcnow, onupdate=datetime.utcnow)
     is_deleted: orm.Mapped[bool] = orm.mapped_column(default=False)
 
-    cooking_time: orm.Mapped[int] = orm.mapped_column(default=0)
+    cooking_time: orm.Mapped[float] = orm.mapped_column(default=0)
```

Comment:
Why float? To represent hours? I'd suggest to have this value as integer and represent seconds, so you could calculate minutes and hours easily